.github/workflows/lint-rust.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     if: github.event.pull_request.draft == false
     strategy:
       matrix:
-        pg_version: [16]
+        pg_version: [16, 17]
```

Comment:
Same here, the workflows which only run on 16 today should only run on 17 instead